ci-operator/step-registry/openshift-extended/test/openshift-extended-test-commands.sh

```diff
@@ -156,6 +156,7 @@ then
     export CLUSTER_ID
 fi
 
+sleep 28800
```

Comment:
Do not use the sleep, it is better if we use this

```
export PATH=/tmp/:$PATH
which extended-platform-tests

echo "WAITING FOR DEBUG..."
while [ ! -f "/tmp/continue" ]
do
    sleep 10
done
```

Reply:
yes I observed that I forgot to remove it. Done.